test/datagateway_api/icat/filters/test_order_filter.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 from typing_extensions import OrderedDict
 
+
```

Comment:
Flake8 isn't complaining about this but the standard gap between import groups is a single line. Could this additional line be removed please? I'm guessing this managed to sneak its way in by accident :)